src/WaitForReact.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { throttle } from 'lodash';
+import changeProps from 'react-change-props';
+import script from './inline-script.raw.js';
+
+const pickScriptState = () =>
+    typeof window !== 'undefined' && Array.isArray(window.__WAIT_FOR_IT__) ? window.__WAIT_FOR_IT__.shift() : undefined;
+
+export default class WaitForReact extends Component {
```

Comment:
So beautiful  👀😁You can resolve this, just an inside joke 👍

Reply:
Hmmm, I'm curious now! What's the inside joke?